src/main/java/rocket/RocketException.java

```diff
@@ -0,0 +1,7 @@
+package rocket;
+
+public class RocketException extends Exception {
```

Comment:
I like the way you defined an exception for this task, with additional sub-classes under it to highlight the problem.